MovieAppPetProject/MovieAppPetProject/Watchlist/WatchlistViewModel.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  WatchlistViewModel.swift
+//  MovieAppPetProject
+//
+//  Created by Sebastian Jacobs on 2024/05/14.
+//
+
+import Foundation
+
+class WatchlistViewModel {
+
+    // MARK: - Variable
+    private let coreDataManager = CoreDataManager()
+
+    // MARK: - Function
+    func fetchAndDisplayWatchlistItems() {
+        let watchlistItems = coreDataManager.getAllWatchlistItems()
+        for item in watchlistItems {
+            print("Original Title: \(item.originalTitle ?? "N/A")")
```

Comment:
Do you still need this print?

Reply:
that print is there just to make sure that the movies added are actually being fetched, it does print so I can remove it when I start displaying the movies with UI

Reply:
Please add a warning to remove the print at a later stage then

Reply:
will do, or I'll just remove it since it is working

Reply:
Yeah I guess you can just remove it. For future you need to get into the habit of using your debugger instead of print statements. Let me know if you would like me to show you

Reply:
I would appreciate it please bro